SourceryTests/SourcerySpec.swift

```diff
@@ -237,40 +237,60 @@ class SourcerySpecTests: QuickSpec {
 
                     it("insert generated code in multiple types") {
                         update(code: """
-                            class Foo {}
```

Comment:
I would have let this test as it is, exposing the insertion of inline code, and would have just added a "Insert auto-inlined code line before body end" test to ensure that the existing insertion code still works.

Reply:
done